src/visualizers/Histogram.ts

```diff
@@ -204,25 +256,137 @@ class FactorHistogramVisualizer extends VisualizerDefault {
-                // Draws text for if the bin size is not 1
+                // Draws text in the case the bin size is not 1
+                binText = (
+                    this.binSize * i
+                    + ' - '
+                    + (this.binSize * (i + 1) - 1)
+                ).toString()
                 this.sketch.text(
-                    this.binSize * i + ' - ' + (this.binSize * (i + 1) - 1),
-                    1 - offsetScalar + binWidth * (i + 1 / 2),
-                    textOffsetScalar * this.sketch.width
+                    binText,
+                    1 - largeOffsetScalar + binWidth * (i + 1 / 2),
+                    smallOffsetScalar * this.sketch.width
                 )
             } else {
-                // Draws text for if the bin size is 1
+                // Draws text in the case the bin size is 1
+                binText = i.toString()
+                this.sketch.text(
+                    binText,
+                    largeOffsetNumber + (binWidth * (i + 1) - binWidth / 2),
+                    smallOffsetScalar * this.sketch.width
+                )
+            }
+        }
+
+        let tickHeight = Math.floor(
+            (0.95 * this.sketch.height) / (height * 5)
+        )
+
+        // Sets the tickHeight to 1 if the calculated value is less than 1
+        if (tickHeight === 0) {
+            tickHeight = 1
+        }
+        // Draws the markings on the Y-axis
+        for (let i = 0; i < 9; i++) {
+            // Draws the tick marks
+            this.sketch.line(
+                (largeOffsetNumber * 3) / 4,
+                this.sketch.height
+                    - largeOffsetNumber
+                    - tickHeight * height * (i + 1),
+                (3 * largeOffsetNumber) / 2,
+                this.sketch.height
+                    - largeOffsetNumber
+                    - tickHeight * height * (i + 1)
+            )
+
+            // Places the numbers on the right side of the axis if
+            // they are 4 digits or more; left side otherwise
+            let tickNudge = 0
+            if (tickHeight > 999) {
+                tickNudge = (3 * largeOffsetNumber) / 2
+            }
+            // Avoid placing text that will get cut off
+            const tickYPosition =
+                this.sketch.height
+                - largeOffsetNumber
+                - tickHeight * height * (i + 1)
+                + (3 * smallOffsetNumber) / 2
+            if (tickYPosition > this.sketch.textAscent()) {
                 this.sketch.text(
-                    i,
-                    (1 - offsetScalar) * this.sketch.width
-                        + (binWidth * (i + 1) - binWidth / 2),
-                    textOffsetScalar * this.sketch.width
+                    tickHeight * (i + 1),
+                    tickNudge,
+                    tickYPosition
                 )
             }
         }
 
-        this.sketch.noLoop()
+        // Draws the box and the text inside the box
```

Comment:
I see you eliminated noLoop altogether; if mouseover is false, is there any reason to loop? Might you still want to call noLoop when mouseover is false?

Reply:
Good point, changed.  This raises the issue though, that "mouseMoved()" and some others aren't working in our setup.  The ideal solution here would be to noLoop() at the end of draw() and only loop() again when mouseMoved().  But that isn't working, so it's an issue to be added/solved.  I added this as #272.

Reply:
I also checked for mouseOver before doing the histogram highlights too, in response to your question above.

Reply:
OK, no more interaction when mouseOver is false.